src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor().toString(), new Random().nextInt(100));
```

Comment:
ColorSupplier and Random should be class level variables
100 is a magic number create constant

Reply:
Thanks, fixed)